circuit_knitting/cutting/cutting_decomposition.py

```diff
@@ -221,14 +229,28 @@ def partition_problem(
     bases = []
     i = 0
     for inst in qpd_circuit.data:
+        if isinstance(inst.operation, SingleQubitQPDGate):
```

Comment:
We probably want to support these for wire cutting, but they are not well-handled as of this PR, so this ValueError is appropriate. We can remove it for the wire cutting PR, if necessary